services/wfm/src/main/java/org/openkilda/wfm/topology/cache/service/CacheService.java

```diff
@@ -101,133 +76,76 @@
      */
     private FlowCache flowCache;
 
-    /**
-     * Network cache cache.
-     */
-    private InMemoryKeyValueState<String, Cache> state;
-
-    private final Auth pathComputerAuth;
-
-    private TopologyContext context;
-    private OutputCollector outputCollector;
-
-    CacheBolt(Auth pathComputerAuth) {
-        this.pathComputerAuth = pathComputerAuth;
-    }
-
-    /**
-     * {@inheritDoc}
-     */
-    @Override
-    public void initState(InMemoryKeyValueState<String, Cache> state) {
-        this.state = state;
-
+    public CacheService(InMemoryKeyValueState<String, Cache> state, PathComputer pathComputer) {
```

Comment:
Don't pass "state" here... if you are going to split "transport" and service layers.

PS This state is not needed even into storm's bolt... but its elimination is not part of this issue.